geometries/GeoDriverCSV.py

```diff
@@ -30,8 +30,8 @@
 #Target = 'HST'
 Target = 'ASPERA'
 galaxy_targ = '9999000'
-instr1 = "ASP_SLIT1"
-instr2 = "ASP_SLIT2"
+instr1 = "ASP_SLIT_0"
+instr2 = "ASP_SLIT_1"
```

Comment:
Kernel set dropped asp_v00.draftE.tf/.ti, using only asp_v000.tf and asp_v000.draftF.ti, which changed the slit instrument names as seen here.